src/main/java/com/spotinst/sdkjava/model/PrivateIpAddresses.java

```diff
@@ -0,0 +1,98 @@
+package com.spotinst.sdkjava.model;
```

Comment:
change this class to API class, BL class is already exists with name of IpAddress and remove builder class from this

Reply:
I Have delete this file and keep PrivateIpAddress